package.json

```diff
@@ -61,7 +61,7 @@
     "select2": "~3.5.2-browserify"
   },
   "scripts": {
-    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/assets/**/*.js || exit 0",
+    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/assets/**/* || exit 0",
```

Comment:
no, eslint only checks .js files unless --ext is used.
Also that does not work, you should see a warning on API.js.